File: BezierPathLearning/BezierPathLearning/ViewController.m
//
//  ViewController.m
//  BezierPathLearning
//
//  Created by Xiangjian Meng on 14-8-21.
//  Copyright (c) 2014年 cn.com.modernmedia. All rights reserved.
//

#import "ViewController.h"
#import "MainView.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad
{
    [super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
    
    MainView *view = [[MainView alloc] initWithFrame:self.view.bounds];
    [self.view addSubview:view];
}

- (void)didReceiveMemoryWarning
{
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: AvatarCaptureView/AvatarCaptureView/ViewController.swift
//
//  ViewController.swift
//  AvatarCaptureView
//
//  Created by Xiangjian Meng on 14/10/23.
//  Copyright (c) 2014年 cn.com.modernmedia. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    var captureView : CaptureView! = nil;
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        
        
        captureView = CaptureView(frame:self.view.bounds);
        captureView.backgroundColor = UIColor.whiteColor();
        self.view.addSubview(captureView);
        
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

    
}

